components/trakt/sources/new-show-or-movie-rated/new-show-or-movie-rated.mjs

```diff
@@ -0,0 +1,29 @@
+import common from "../common/common.mjs";
+
+export default {
+  ...common,
+  name: "New Show Or Movie Rated",
+  version: "0.0.1",
+  key: "trakt-new-show-or-movie-rated",
+  description: "Emit new event on each new rated show or movie.",
```

Comment:
We should link the docs of the endpoints used for polling, as Alyson mentioned in the previous review.